### ISSUE TITLE
Modern Skin, Login Page, Chromium Browser

### ISSUE THREAD

**Issue by interduo** [Cacti/cacti#120]
Cannot login

after clicking login, nothing happens and the button is grey.

**Comment by cigamit** [comment]
Likely a JavaScript error, look into your browsers debug window for the error and post it.

**Comment by shavenne** [comment]
Had that problem too. Nothing is coming into the console of Chrome.
When I set the global theme to Classic and the user theme to modern it works. But there I had also a problem with buttons then. I couldn't save at a graph template. Nothing happened after clicking the button.

**Comment by cigamit** [comment]
Did you press f5 from the login screen to reload the browsers cache?  Just want to see if that makes a difference.

**Comment by shavenne** [comment]
Yes. Also tried incognito mode.
Same problem on my android smartphone btw.

**Comment by cigamit** [comment]
Should be resolved now.  Make sure you press f5 to reload layout.js from the website instead of the cached copy.

**Comment by dabeani** [comment]
not, same in safari and chrome. also reloading does not help. with inkognito the login is also not workling. The Login button goes in disable mode after klicking... but thats it - i cannot see any error log or javascript error. So, no login ATM.

**Comment by shavenne** [comment]
Same here. Did not work for me either :/

**Comment by cigamit** [comment]
Ok, do a few things:
1) Install firebug for Firefox, enable and display the console
2) edit /etc/php.ini and set error_log = /tmp/php_errors.log
3) restart apache
4) attempt to login again

We would be looking for a post back to the web server when you click the login button.  You should be able to see that from either the Net or the Console.  Might be easier from the Net panel.  We would be looking for both the fact that it sent something to the server, and the response.

**Comment by cigamit** [comment]
Okay, I've reproduced it in Chrome.  Looking into it now.

**Comment by cigamit** [comment]
Once you reproduce something, it's easier to fix.  Resolved.  Chrome behaves differently than Firefox and IE.  That's the first time I've said something like that in a long while.

**Comment by smiles1969** [comment]
I've got the same issue with Chrome and Dark theme, even after the update. Cant log in.. Works fine in exploder.

**Comment by interduo** [comment]
Thank You :)

**Comment by cigamit** [comment]
F5 to reload things from Cacti.

**Comment by smiles1969** [comment]
hitting f5 does allow me to log in. Also.. There are a few forms on plugins that have the same behavior, I have to refresh before saving, or the form button will not save the data.

**Comment by ronytomen** [comment]
Which browser and version of said browser are you using?

**Comment by smiles1969** [comment]
Chrome Version 55.0.2883.87 m

**Comment by cigamit** [comment]
You have to be experiencing a javascript error as I have tested this Chrome version on Windows7 without errors.  Can you please open the Developer tools in Chrome, and set the console to log javascript errors, and then post what errors are coming through the browser.  This would be the only thing that I can think of that would force you to do a refresh before being able to actually submit.

Also, please tell us what page your are in when you can not click a button and where you had come from before you had gotten there.

**Comment by smiles1969** [comment]
I am not seeing any js errors in the console window, only navigation from page to page. 

As an example. I am trying to add new devices in the routerconfigs plugin, coming from Utilities on left pane of main console window. I do see missing telnet and feedback images in the console, but I didn't figure that was related, as they were there before clicking on add device and getting to the device add page itself.

**Comment by cigamit** [comment]
Okay, let's take Routerconfigs out of the equation for now.  Assuming you are not navigating to any plugins, can you reproduce this?  It could simply be a bug in Routerconfigs.  We need to be able to separate the plugins from the main GUI for this bug.  Routerconfigs is a bad one to choose as it has had come migration testing, but little things like some form not functioning may have been missed in the migration.  To make forms more reliable, change the doctype in the following pages: top_header.php, top_graph_header.php, and top_general_header.php from:

 <!DOCTYPE html> 

to:

 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"> 

And see if those broken forms start working.

**Comment by smiles1969** [comment]
I was seeing the same behavior on the login screen that I was seeing on plugins, which is why I mentioned it on this issue. As of now, I am not having a problem logging in to Cacti. So no, I can not reproduce this currently.

As for routerconfigs, I've changed the doctypes on those files, but am not seeing any difference in behavior.

I had previously opened an issue on Dec 13th regarding the form save button for the routerconfigs plugin on it's own git repo. I also noticed you are not listed on it's watch list so you probably didn't get notified when I did.
